core/src/test/kotlin/org/stellar/anchor/sep31/Sep31ServiceTest.kt

```diff
@@ -51,13 +49,13 @@ class Sep31ServiceTest {
     val gson: Gson = GsonUtils.getInstance()
 
     private const val stellarUSDC =
-      "stellar:USDC:GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP"
+        "stellar:USDC:GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP"
```

Comment:
👀  Can you re-format the code, so we don't add these whitespaces? You can do so by running `./gradlew spotlessApply`.

It should be automatically called in the pre-commit... not sure what happened 🤔

Reply:
Yeah... that's weird that the pre-commit hook did not happen. I will check later. 

`./gradlew spotlessApply` is now executed.